components/sidebar/index.css

```diff
@@ -0,0 +1,28 @@
+.close {
+  width: 32px;
+  height: 32px;
+  opacity: 1;
+  justify-self: flex-end;
+  position: relative;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+.close:hover {
+  opacity: 0.3
+}
+.close:before, .close:after {
```

Comment:
You could use an icon but it's up to you.

Reply:
This approach doesn't require image download. None that it matters. Maybe I will switch to icon later on.